packages/fx-core/src/common/samples.ts

```diff
@@ -34,7 +34,7 @@ class SampleProvider {
           configuration: sample.configuration,
           link: sample.packageLink ?? sampleConfig.defaultPackageLink,
           suggested: sample.suggested,
-          url: sample.url ?? sampleConfig.baseUrl,
+          url: sample.relativePath ? sample.url : sample.url ?? sampleConfig.baseUrl,
```

Comment:
Looks like the logic is same with old one. `relativePath` is not used?

Reply:
relativePath is used to determine whether it's external sample.

Reply:
`relativePath` is used for external sample which will use its own url instead of base version.